modules/openapi-generator/src/test/resources/3_0/petstore.yaml

```diff
@@ -283,6 +283,10 @@ paths:
                   description: file to upload
                   type: string
                   format: binary
+                secondFile:
+                  description: another file to upload
+                  type: string
+                  format: binary
```

Comment:
Do we want to add it to this file? It will affect all clients.

Reply:
I went ahead and removed it. Looking through active and closed bugs, it looks like other clients have been impacted by this issue to.